assets/js/users/private.js

```diff
@@ -42,10 +42,11 @@ Alpine.data("UserGraphs", () => ({
 
     let data = [];
     for (const property in breakdown) {
+      let percent_result = (breakdown[property] / categories.length) * 100;
       data.push({
         "name": property,
         "count": breakdown[property],
-        "percent": (breakdown[property] / categories.length) * 100,
+        "percent": Number(percent_result.toFixed(2)),
```

Comment:
I don't think you need to call toFixed here actually? Becuase percent_result should already be a number.

```
> 98.967.toFixed(2)
'98.97'
```